File: Script Utility Toolbox/javascript_scripts/publish_bulk_message_to_mqtt.js
const mqtt = require('mqtt');
const { v4: uuidv4 } = require('uuid');

const brokerUrl = 'brokerUrl'; // Adjust the base path accordingly
const options = {
  clientId: 'clientId',
  transport: 'transport',
  username: 'username',
  password: 'password',
};

const client = mqtt.connect(brokerUrl, options);

client.on('connect', function () {
  console.log('[green] connected with result code');
  client.subscribe('topic', function (err) {
    if (!err) {
      console.log('[green] Subscribed to call_events');
    } else {
      console.error('[red] Error subscribing to call_events:', err);
    }
  });
});

client.on('message', function (topic, message) {
  console.log(`${topic} ${message.toString()}`);
});

const groupId = 'groupId';
const myMessages = ['Hello', 'Samiul'];
const id = ['id'];

try {
  for (let i = 0; i < 10; i++) {
    const aiGuess = Math.floor(Math.random() * 2);

    const ack = {
      // payload
    };

    const value = {
            // payload

    };

    const message = {
            // payload

    };

    const formattedMessage = JSON.stringify(message);

    console.log('#### ', i);
    
    client.publish('topic', formattedMessage);
  }
} catch (error) {
  console.error('[red] An error occurred:', error);
  client.end();
}

File: Script Utility Toolbox/python_scripts/publish_bulk_message_to_mqtt.py
import paho.mqtt.client as mqtt
import time
import uuid
import json
import random 
from random import randrange

def on_connect(client, obj, flags, rc):
    if rc == 0:
        print(f"[green] connected with result code")
    else:
        print(f"[red] Bad connection returned code = ", rc)

def on_message(client, obj, msg):
    print(f"{msg.topic} {str(msg.payload)}")

Connected = False   # global variable for the state of the connection

# Set use_websockets to True and use the ws:// or wss:// prefix in the broker URL
broker_url = "broker_url"  # Adjust the base path accordingly
client = mqtt.Client(client_id="client_id", transport='transport')  # Set transport to websockets
client.username_pw_set("username_pw_set", "pass")
client.on_connect = on_connect
client.on_message = on_message
client.connect(broker_url, keepalive=60, bind_address="")

client.subscribe("topic")
time.sleep(1)

groupId = "groupId"
my_list = ["Hello", "World"]  # Adjust the list of messages accordingly
id = ["id"]

try:
    for i in range(10):
        aiGuess = randrange(1)

        ack = {}

        value = {}

        message = {}

        formatedMessage = json.dumps(message)

        print("#### ", i)

        client.publish("topic", formatedMessage)

except KeyboardInterrupt:
    client.disconnect()
    client.loop_stop()
client.loop_forever()
